### PR TITLE
[upgrade] switch to debug logging on upgrade

### PR DESCRIPTION
**Pull request by MorrisJobke**
- resets afterwards
- adds output about the previous log level

cc @PVince81 @nickvergessen @LukasReschke @karlitschek As discussed multiple times.

I would like to backport this down to stable7.

### CODE REVIEW

File: core/command/upgrade.php [hunk 1]
```diff
@@ -177,6 +177,12 @@ function ($success) use($output, $updateStepEnabled, $self) {
 			$updater->listen('\OC\Updater', 'failure', function ($message) use($output, $self) {
 				$output->writeln("<error>$message</error>");
 			});
+			$updater->listen('\OC\Updater', 'setDebugLogLevel', function ($logLevel, $logLevelName) use($output) {
```
Comment (PVince81) [inline core/command/upgrade.php]: Maybe it's time to introduce a new generic "message" event ?

**Comment by MorrisJobke** [comment]
cc @DeepDiver1975

**Comment by karlitschek** [comment]
yes. probably a good idea :+1:

File: core/command/upgrade.php (hunk 1 above)
Comment (MorrisJobke) [inline core/command/upgrade.php]: Will do for 9.0

**Comment by PVince81** [comment]
:+1:

**Comment by MorrisJobke** [comment]
Backport PRs created :)

**Pull request merged**